### ISSUE TITLE
Tower Activation does not lock player in

### ISSUE THREAD

**Issue by Ehseezed** [TheyCallMeXander/AM2R-SA-X-Duels#37]
The tower activation room does not properly lock in players who enter via the power bomb door.

**Comment by ANXvariable** [comment]
If I understand correctly, this is about what you saw when I, as SA-X, entered Tower Activation and Activated Tower, then reloaded the room and exited through the left door? In which case, that may not be a universal issue. That match had the Experimental Boss Switch enabled, which disables bosses like Tester, and normally defeating Tester would unlock that left door anyway. There is the possibility that Xander still wants you locked into the Tower even with bosses disabled, though. Not sure.
TLDR the door being opened is expected behavior when Tester doesn't exist

**Comment by ANXvariable** [comment]
i have been informed of the context of the opening of this issue on github

**Comment by TheyCallMeXander** [comment]
Works as intended
Note: do not toggle bosses off after tower activation: https://www.youtube.com/watch?v=gBHyJIoeCf0
